From: Jose Luis Rivero <jrivero@osrfoundation.org>
Date: Tue, 14 Apr 2020 16:31:14 +0200
Subject: Use system packaged roboto fonts

Last-Update: 2019-02-26
---
 gazebo/gui/resources.qrc | 25 ++++++++++++-------------
 1 file changed, 12 insertions(+), 13 deletions(-)

diff --git a/gazebo/gui/resources.qrc b/gazebo/gui/resources.qrc
index 7a7a6da..934d1d6 100644
--- a/gazebo/gui/resources.qrc
+++ b/gazebo/gui/resources.qrc
@@ -102,18 +102,17 @@
   <file>images/avi.svg</file>
   <file>images/v4l.svg</file>
   <file>images/video_camera.svg</file>
-
-  <file>fonts/Roboto-Black.ttf</file>
-  <file>fonts/Roboto-BlackItalic.ttf</file>
-  <file>fonts/Roboto-Bold.ttf</file>
-  <file>fonts/Roboto-BoldItalic.ttf</file>
-  <file>fonts/Roboto-Italic.ttf</file>
-  <file>fonts/Roboto-Light.ttf</file>
-  <file>fonts/Roboto-LightItalic.ttf</file>
-  <file>fonts/Roboto-Medium.ttf</file>
-  <file>fonts/Roboto-MediumItalic.ttf</file>
-  <file>fonts/Roboto-Regular.ttf</file>
-  <file>fonts/Roboto-Thin.ttf</file>
-  <file>fonts/Roboto-ThinItalic.ttf</file>
+  <file>/usr/share/fonts/truetype/roboto/unhinted/RobotoTTF/Roboto-Black.ttf</file>
+  <file>/usr/share/fonts/truetype/roboto/unhinted/RobotoTTF/Roboto-BlackItalic.ttf</file>
+  <file>/usr/share/fonts/truetype/roboto/unhinted/RobotoTTF/Roboto-Bold.ttf</file>
+  <file>/usr/share/fonts/truetype/roboto/unhinted/RobotoTTF/Roboto-BoldItalic.ttf</file>
+  <file>/usr/share/fonts/truetype/roboto/unhinted/RobotoTTF/Roboto-Italic.ttf</file>
+  <file>/usr/share/fonts/truetype/roboto/unhinted/RobotoTTF/Roboto-Light.ttf</file>
+  <file>/usr/share/fonts/truetype/roboto/unhinted/RobotoTTF/Roboto-LightItalic.ttf</file>
+  <file>/usr/share/fonts/truetype/roboto/unhinted/RobotoTTF/Roboto-Medium.ttf</file>
+  <file>/usr/share/fonts/truetype/roboto/unhinted/RobotoTTF/Roboto-MediumItalic.ttf</file>
+  <file>/usr/share/fonts/truetype/roboto/unhinted/RobotoTTF/Roboto-Regular.ttf</file>
+  <file>/usr/share/fonts/truetype/roboto/unhinted/RobotoTTF/Roboto-Thin.ttf</file>
+  <file>/usr/share/fonts/truetype/roboto/unhinted/RobotoTTF/Roboto-ThinItalic.ttf</file>
 </qresource>
 </RCC>
