Description: respect given CMAKE_SHARED_LINKER_FLAGS
Author: Jose Luis Rivero <jrivero@osrfoundation.org>
Forwarded: https://bitbucket.org/osrf/gazebo/pull-request/1122

--- gazebo-3.0.0.orig/deps/opende/CMakeLists.txt
+++ gazebo-3.0.0/deps/opende/CMakeLists.txt
@@ -243,7 +243,7 @@ if (SSE2_FOUND OR SSE3_FOUND OR SSSE3_FO
   set (CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -DSSE")
 endif()
 
-set (CMAKE_SHARED_LINKER_FLAGS "${CMAKE_LINK_FLAGS_${CMAKE_BUILD_TYPE}} -MF -MT -fno-strict-aliasing -DPIC ")
+set (CMAKE_SHARED_LINKER_FLAGS "${CMAKE_SHARED_LINKER_FLAGS} ${CMAKE_LINK_FLAGS_${CMAKE_BUILD_TYPE}} -MF -MT -fno-strict-aliasing -DPIC ")
 
 gz_add_library(gazebo_ode ${sources})
 target_link_libraries(gazebo_ode
