Description: Fix comparison to install manpages
Author: Jose Luis Rivero <jrivero@osrfoundation.org>
Forwarded: https://bitbucket.org/osrf/gazebo/pull-request/1123

--- gazebo-3.0.0.orig/tools/CMakeLists.txt
+++ gazebo-3.0.0/tools/CMakeLists.txt
@@ -43,7 +43,7 @@ add_executable(gzfactory gzfactory.cc)
 target_link_libraries(gzfactory gazebo_common gazebo_transport pthread)
 gz_install_executable(gzfactory)
 
-if (${GAZEBO_VERSION} VERSION_GREATER 3.0)
+if (${GAZEBO_VERSION} VERSION_LESS 4.0)
   message (WARNING
     "Installing deprecated manpages. This should be removed after Gazebo 3.0")
   manpage(gzfactory 1)
