Description: Use system packaged gtest
Author: Jose Luis Rivero <jrivero@osrfoundation.org>
Last-Update: 2015-08-02

diff --git a/test/CMakeLists.txt b/test/CMakeLists.txt
index 975bed1..4873a7a 100644
--- a/test/CMakeLists.txt
+++ b/test/CMakeLists.txt
@@ -1,4 +1,6 @@
 include_directories (
+  /usr/include
+  /usr/src/gtest
   ${ODE_INCLUDE_DIRS}
   ${OPENGL_INCLUDE_DIR}
   ${OGRE_INCLUDE_DIRS}
@@ -24,8 +26,8 @@ endif()
 configure_file (test_config.h.in ${PROJECT_BINARY_DIR}/test_config.h)
 
 # Build gtest
-add_library(gtest STATIC gtest/src/gtest-all.cc)
-add_library(gtest_main STATIC gtest/src/gtest_main.cc)
+add_library(gtest STATIC /usr/src/gtest/src/gtest-all.cc)
+add_library(gtest_main STATIC /usr/src/gtest/src/gtest_main.cc)
 target_link_libraries(gtest_main gtest)
 set(GTEST_LIBRARY "${PROJECT_BINARY_DIR}/test/libgtest.a")
 set(GTEST_MAIN_LIBRARY "${PROJECT_BINARY_DIR}/test/libgtest_main.a")
