Description: do not unset CMAKE_C*_FLAGS
Author: Jose Luis Rivero <jrivero@osrfoundation.org>
Forwarded: no

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 63ea6f7..d5406f9 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -181,8 +181,6 @@ endif()
 
 #####################################
 # Handle CFlags
-unset (CMAKE_C_FLAGS_ALL CACHE)
-unset (CMAKE_CXX_FLAGS CACHE)
 
 # USE_HOST_CFLAGS (default TRUE)
 # Will check building host machine for proper cflags
