Description: update ann copyrigth 
Author: Jose Luis Rivero <jrivero@osrfoundation.org>
Forwarded: https://bitbucket.org/osrf/gazebo/pull-request/1465


diff -r 8279a82b2a9f deps/ann/include/ann/ANN.h
--- a/deps/ann/include/ann/ANN.h	Fri Feb 27 01:57:15 2015 +0100
+++ b/deps/ann/include/ann/ANN.h	Fri Feb 27 02:02:40 2015 +0100
@@ -5,7 +5,7 @@
 //					neighbor searching.
 // Last modified:	01/27/10 (Version 1.1.2)
 //----------------------------------------------------------------------
-// Copyright (c) 1997-2010 University of Maryland and Sunil Arya and
+// Copyright (c) 1997-2015 University of Maryland and Sunil Arya and
 // David Mount.  All Rights Reserved.
 // 
 // This software and related documentation is part of the Approximate
diff -r 8279a82b2a9f deps/ann/include/ann/ANNx.h
--- a/deps/ann/include/ann/ANNx.h	Fri Feb 27 01:57:15 2015 +0100
+++ b/deps/ann/include/ann/ANNx.h	Fri Feb 27 02:02:40 2015 +0100
@@ -11,7 +11,7 @@
 //
 //	Typical users of ANN should not need to access this file.
 //----------------------------------------------------------------------
-// Copyright (c) 1997-2010 University of Maryland and Sunil Arya and
+// Copyright (c) 1997-2015 University of Maryland and Sunil Arya and
 // David Mount.  All Rights Reserved.
 // 
 // This software and related documentation is part of the Approximate
