Description: Remove unknown roff commands
Author: Jose Luis Rivero <jrivero@osrfoundation.org>
Forwarded: no

diff --git a/tools/gz.1.roff b/tools/gz.1.roff
index 1cfb9a4..4fc0c81 100644
--- a/tools/gz.1.roff
+++ b/tools/gz.1.roff
@@ -6,9 +6,7 @@
 .SH "SYNPOSIS"
 \fBgz\fR \fIcommand\fR [option]\.\.\. [argument]\.\.\.
 .
-.UNINDENT
 .SH COMMANDS
-.UNINDENT
 .SS camera
 .sp
 .nf
@@ -25,7 +23,6 @@ A camera name is required.
 
 .sp
 Options:
-.INDENT 0.0
 .TP
 .B \-h, \-\-help
 .
@@ -46,7 +43,6 @@ List all cameras
 .B \-f, \-\-follow\fR=\fIarg\fR
 .
 Model to follow.
-.UNINDENT
 .SS debug
 .sp
 .nf
@@ -61,7 +57,6 @@ return the completion list for a given command.
 
 .sp
 Options:
-.INDENT 0.0
 .TP
 .B \-h, \-\-help
 .
@@ -70,7 +65,6 @@ Print this help message
 .B \-o, \-\-option\fR=\fIarg\fR
 .
 Show the command options.
-.UNINDENT
 .SS help
 .sp
 .nf
@@ -84,12 +78,10 @@ Output information about a gz command.
 
 .sp
 Options:
-.INDENT 0.0
 .TP
 .B \-h, \-\-help
 .
 Print this help message
-.UNINDENT
 .SS joint
 .sp
 .nf
@@ -106,7 +98,6 @@ A model name and joint name are required.
 
 .sp
 Options:
-.INDENT 0.0
 .TP
 .B \-h, \-\-help
 .
@@ -163,7 +154,6 @@ Velocity integral gain.
 .B \-\-vel\-d\fR=\fIarg\fR
 .
 Velocity differential gain.
-.UNINDENT
 .SS log
 .sp
 .nf
@@ -179,7 +169,6 @@ an active Gazebo server.
 
 .sp
 Options:
-.INDENT 0.0
 .TP
 .B \-h, \-\-help
 .
@@ -224,7 +213,6 @@ Path to a log file.
 .B \-\-filter\fR=\fIarg\fR
 .
 Filter output. Valid only for the echo and step commands
-.UNINDENT
 .SS model
 .sp
 .nf
@@ -241,7 +229,6 @@ will be used.
 
 .sp
 Options:
-.INDENT 0.0
 .TP
 .B \-h, \-\-help
 .
@@ -286,7 +273,6 @@ Output model pose as a space separated 6\-tuple: x y z roll pitch yaw.
 .B \-P, \-\-pose\-P arg pitch in radians.
 .TP
 .B \-Y, \-\-pose\-Y arg yaw in radians.
-.UNINDENT
 .SS physics
 .sp
 .nf
@@ -302,7 +288,6 @@ the first world found on the Gazebo master will be used.
 
 .sp
 Options:
-.INDENT 0.0
 .TP
 .B \-h, \-\-help
 .
@@ -331,7 +316,6 @@ Target real\-time update rate.
 .B \-o, \-\-profile\fR=\fIarg\fR
 .
 Preset physics profile.
-.UNINDENT
 .SS sdf
 .sp
 .nf
@@ -347,7 +331,6 @@ SDF for use with other options.
 
 .sp
 Options:
-.INDENT 0.0
 .TP
 .B \-h, \-\-help
 .
@@ -376,7 +359,6 @@ Version of SDF to use with other options.
 .B \-p, \-\-print\fR=\fIarg\fR
 .
 Print arg, useful for debugging and as a conversion tool.
-.UNINDENT
 .SS stats
 .sp
 .nf
@@ -392,7 +374,6 @@ the Gazebo master will be used.
 
 .sp
 Options:
-.INDENT 0.0
 .TP
 .B \-h, \-\-help
 .
@@ -409,7 +390,6 @@ Duration (seconds) to run.
 .B \-p, \-\-plot
 .
 Output comma\-separated values, useful for processing and plotting.
-.UNINDENT
 .SS topic
 .sp
 .nf
@@ -425,7 +405,6 @@ the Gazebo master will be used.
 
 .sp
 Options:
-.INDENT 0.0
 .TP
 .B \-h, \-\-help
 .
@@ -466,7 +445,6 @@ Output data from echo without formatting.
 .B \-d, \-\-duration\fR=\fIarg\fR
 .
 Duration (seconds) to run. Applicable with echo, hz, and bw
-.UNINDENT
 .SS world
 .sp
 .nf
@@ -482,7 +460,6 @@ the first world found on the Gazebo master will be used.
 
 .sp
 Options:
-.INDENT 0.0
 .TP
 .B \-h, \-\-help
 .
