Description: PDF generation is broken
Author: Jose Luis Rivero <jrivero@osrfoundation.org>
Bug: https://bitbucket.org/osrf/gazebo/issue/1141
Forwarded: not-needed

--- gazebo-3.0.0.orig/doc/CMakeLists.txt
+++ gazebo-3.0.0/doc/CMakeLists.txt
@@ -16,9 +16,6 @@ if (DOXYGEN_FOUND)
             ${CMAKE_BINARY_DIR}/doxygen/html
     COMMAND cp ${CMAKE_SOURCE_DIR}/doc/search.js
             ${CMAKE_BINARY_DIR}/doxygen/html/search
-    COMMAND make -C ${CMAKE_BINARY_DIR}/doxygen/latex
-    COMMAND mv ${CMAKE_BINARY_DIR}/doxygen/latex/refman.pdf
-            ${CMAKE_BINARY_DIR}/doxygen/latex/gazebo-${GAZEBO_VERSION_FULL}.pdf
 
     # Generate the messages documentation
     COMMAND doxygen ${CMAKE_BINARY_DIR}/gazebo_messages.dox
