Description: remove deprecated gzmodel_create script
Author: Jose Luis Rivero <jrivero@osrfoundation.org>
Bug: https://bitbucket.org/osrf/gazebo/pull-request/1130

--- gazebo-3.0.0.orig/tools/CMakeLists.txt
+++ gazebo-3.0.0/tools/CMakeLists.txt
@@ -61,7 +61,5 @@ gz_install_executable(gz)
 
 roffman(gz 1)
 
-install (PROGRAMS gzmodel_create gzmodel_create DESTINATION ${BIN_INSTALL_DIR})
-
 install (PROGRAMS gzprop DESTINATION ${BIN_INSTALL_DIR})
 manpage(gzprop 1)
--- gazebo-3.0.0.orig/tools/gzmodel_create
+++ /dev/null
@@ -1,27 +0,0 @@
-#!/bin/sh
-
-manifest="<?xml version='1.0'?>
-<model>
-  <name>my_model</name>
-  <version>1.0</version>
-  <sdf>model.sdf</sdf>
-
-  <author>
-    <name>my_name</name>
-    <email>me@my.email</email>
-  </author>
-  
-  <description>
-  A description of my model
-  </description>
-</model>";
-
-if [ $1 ]
-then
-  mkdir $1
-  echo "$manifest" > $1/model.config;
-else
-  echo "Enter a directory to create for the model."
-  echo "Usage:"
-  echo "  gzmodel_create [directory]"
-fi
