From: Jose Luis Rivero <jrivero@osrfoundation.org>
Date: Wed 26 May 2021 04:59:18 PM UTC
Subject: Inclued build of gazebo_test_fixture when running make install
Fordwarded:


---

diff --git a/gazebo/test/CMakeLists.txt b/gazebo/test/CMakeLists.txt
index 0f71e8ba..dbe16528 100644
--- a/gazebo/test/CMakeLists.txt
+++ b/gazebo/test/CMakeLists.txt
@@ -8,3 +8,5 @@ gz_install_includes("test"
   ServerFixture.hh
   ${PROJECT_BINARY_DIR}/test_config.h
 )
+
+set_target_properties(gazebo_test_fixture PROPERTIES DEPENDS ALL)
