Description: Replace deprecated FFmpeg API
Author: Andreas Cadhalpun <Andreas.Cadhalpun@googlemail.com>
Author: Jose Luis Rivero <jrivero@osrfoundation.org>
Last-Update: <2016-07-15>

diff --git a/gazebo/common/Video.cc b/gazebo/common/Video.cc
index 1a12e92..b53b9b0 100644
--- a/gazebo/common/Video.cc
+++ b/gazebo/common/Video.cc
@@ -67,7 +67,7 @@ void Video::Cleanup()
 {
 #ifdef HAVE_FFMPEG
   // Free the YUV frame
-  av_free(this->avFrame);
+  av_frame_free(this->avFrame);
 
   // Close the video file
   avformat_close_input(&this->formatCtx);
@@ -75,7 +75,7 @@ void Video::Cleanup()
   // Close the codec
   avcodec_close(this->codecCtx);
 
-  av_free(avFrameDst);
+  av_frame_free(avFrameDst);
 #endif
 }
