Description: Replace deprecated FFmpeg API
Author: Andreas Cadhalpun <Andreas.Cadhalpun@googlemail.com>
Last-Update: <2015-11-02>

--- gazebo-6.5.0+dfsg.orig/gazebo/common/Video.cc
+++ gazebo-6.5.0+dfsg/gazebo/common/Video.cc
@@ -70,7 +70,7 @@ void Video::Cleanup()
 {
 #ifdef HAVE_FFMPEG
   // Free the YUV frame
-  av_free(this->avFrame);
+  av_frame_free(&this->avFrame);
 
   // Close the video file
   avformat_close_input(&this->formatCtx);
@@ -147,7 +147,7 @@ bool Video::Load(const std::string &_fil
     return false;
   }
 
-  avpicture_alloc(this->pic, PIX_FMT_RGB24, this->codecCtx->width,
+  avpicture_alloc(this->pic, AV_PIX_FMT_RGB24, this->codecCtx->width,
                   this->codecCtx->height);
 
   this->swsCtx = sws_getContext(
@@ -156,7 +156,7 @@ bool Video::Load(const std::string &_fil
       this->codecCtx->pix_fmt,
       this->codecCtx->width,
       this->codecCtx->height,
-      PIX_FMT_RGB24,
+      AV_PIX_FMT_RGB24,
       SWS_BICUBIC, NULL, NULL, NULL);
 
   if (this->swsCtx == NULL)
